### DISCUSSION TITLE
Icon on the filename component

### DISCUSSION THREAD

**Discussion by zer09** [nvim-lualine/lualine.nvim#607] [Q&A] [answered]
How can I make the icon is much closer to the filename?
![Screenshot_20220314_000806](https://user-images.githubusercontent.com/780780/158069314-ae01acf1-a3b5-41ee-bd89-db208dff397a.png)

**Comment by fitrh** [comment, marked as answer]
Set the components right padding to 0

```lua
{
    "filetype",
    padding = { left = 1, right = 0 },
    colored = true,
    icon_only = true,
}
```

**Reply by zer09** [reply]
Thank you so much
